ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "caseTitleWithHWFFeeNumber",
```

Comment:
Use caseTitleWithHWFClaimFeeNumber to be consistent pls

Reply:
updated